wrappers/IWearLogger/src/IWearLogger.cpp

```diff
@@ -693,6 +983,47 @@ bool IWearLogger::attach(yarp::dev::PolyDriver* poly)
     return true;
 }
 
+bool IWearLogger::impl::configureMatlabBufferManager(const std::string& sensorName, const size_t& channelSize)
+{
+    auto channelName = convertSensorNameToValidMatlabVarName(sensorName);
+    wearable2MatlabNameLookup[sensorName] = channelName;
+
+    bool ok = bufferManager.addChannel({channelName, {channelSize, 1}});
+
+    if (!ok)
+    {
+        yError() << logPrefix << " matlab buffer manager configuration failed for " << sensorName;
+        return false;
+    }
+
+    return true;
+}
+
+bool IWearLogger::impl::configureYarpBufferManager(const std::string &sensorName)
+{
+    auto portName = convertSensorNameToValidYarpPortName(sensorName);
+
+    std::unique_ptr<yarp::os::BufferedPort<yarp::os::Bottle>> port = std::make_unique<yarp::os::BufferedPort<yarp::os::Bottle>>();
```

Comment:
```suggestion
    auto port = std::make_unique<yarp::os::BufferedPort<yarp::os::Bottle>>();
```

Reply:
Done in https://github.com/robotology/wearables/pull/123/commits/afacfdf1c362f7bf87ef28f853f25037a6a3ca8a